samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/DocumentImportController.cs

```diff
@@ -262,10 +292,36 @@ private SupportedFileType GetFileType(string fileName)
         {
             ".txt" => SupportedFileType.Txt,
             ".pdf" => SupportedFileType.Pdf,
+            ".jpg" => SupportedFileType.Jpg,
+            ".jpeg" => SupportedFileType.Jpg,
+            ".png" => SupportedFileType.Png,
+            ".tif" => SupportedFileType.Tiff,
+            ".tiff" => SupportedFileType.Tiff,
             _ => throw new ArgumentOutOfRangeException($"Unsupported file type: {extension}"),
         };
     }
 
+    /// <summary>
+    /// Reads the text content from an image file.
+    /// </summary>
+    /// <param name="file">An IFormFile object.</param>
+    /// <returns>A string of the content of the file.</returns>
+    private async Task<string> ReadTextFromImageFileAsync(IFormFile file)
+    {
+        await using (var ms = new MemoryStream())
+        {
+            await file.CopyToAsync(ms);
+            var fileBytes = ms.ToArray();
+            await using var imgStream = new MemoryStream(fileBytes);
+
+            using var engine = new TesseractEngine(_tesseractDataPath, _tesseractOptions.Language, EngineMode.Default); // use the appropriate language model
```

Comment:
instead of creating a new engine everytime an image is uploaded, which seems it could be potentially expensive depending on the size/number of files in the `tessdata` folder, can we just create the engine once if the folder exists?

Reply:
That's fair - The same engine would be used per upload, but it would be re-instantiated upon the next upload.  I've made updates to make this a singleton.